### PR TITLE
zipsplit: add page

### PR DESCRIPTION
**Pull request by pg99696**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/zipsplit.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# zipsplit
+
+> Zipsplit reads a zipfile and splits it into smaller zipfiles.
+> More information: <https://linux.die.net/man/1/zipsplit>.
+
+- Split zipfile into pieces that are no larger than 1 megabyte:
+
+`zipsplit -n 1048576 archive.zip`
```
Comment (kbdharun) [inline pages/linux/zipsplit.md:8]:
```suggestion
`zipsplit -n {{size}} {{path/to/archive.zip}}`
```

File: pages/linux/zipsplit.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# zipsplit
+
+> Zipsplit reads a zipfile and splits it into smaller zipfiles.
+> More information: <https://linux.die.net/man/1/zipsplit>.
```
Comment (kbdharun) [inline pages/linux/zipsplit.md:4]:
```suggestion
> More information: <https://manned.org/zipsplit>.
```
We prefer manned when official documentation isn't available

File: pages/linux/zipsplit.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+- Split zipfile into pieces that are no larger than 1 megabyte:
```
Comment (kbdharun) [inline pages/linux/zipsplit.md:6]:
```suggestion
- Split zipfile into pieces that are no larger than a particular [n]size:
```

File: pages/linux/zipsplit.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+`zipsplit -n 1048576 archive.zip`
+
+- Pause between the creation of each split file:
+
+`zipsplit -p -n 2097152 archive.zip`
```
Comment (kbdharun) [inline pages/linux/zipsplit.md:12]:
```suggestion
`zipsplit -p -n {{size}} {{path/to/archive.zip}}`
```

File: pages/linux/zipsplit.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
+# zipsplit
+
+> Zipsplit reads a zipfile and splits it into smaller zipfiles.
+> More information: <https://linux.die.net/man/1/zipsplit>.
+
+- Split zipfile into pieces that are no larger than 1 megabyte:
+
+`zipsplit -n 1048576 archive.zip`
+
+- Pause between the creation of each split file:
```
Comment (kbdharun) [inline pages/linux/zipsplit.md:10]:
```suggestion
- [p]ause between the creation of each split zipfile:
```
We use mnemonics for single-character flags whenever possible.

File: pages/linux/zipsplit.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+
+`zipsplit -p -n 2097152 archive.zip`
+
+- Write the split zipfiles into the directory /archive:
```
Comment (kbdharun) [inline pages/linux/zipsplit.md:14]:
```suggestion
- Output the split zipfiles into the `archive` directory :
```

File: pages/linux/zipsplit.md [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+
+`zipsplit -b /archive -n 2097152 archive.zip`
```
Comment (kbdharun) [inline pages/linux/zipsplit.md:16]:
```suggestion
`zipsplit -b {{archive}} -n {{size}} {{path/to/archive.zip}}`
```

File: pages/linux/zipsplit.md [hunk 8]
```diff
@@ -0,0 +1,16 @@
+# zipsplit
+
+> Zipsplit reads a zipfile and splits it into smaller zipfiles.
```
Comment (kbdharun) [inline pages/linux/zipsplit.md:3]:
```suggestion
> Read a zipfile and split it into smaller zipfiles.
```

**Review by kbdharun — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Comment by pg99696** [comment]
@kbdharun, can you please add label 'hactoberfest-accepted' on PR?

**Comment by kbdharun** [comment]
> @kbdharun, can you please add label 'hactoberfest-accepted' on PR?

This PR will automatically get reflected in your profile without the accepted label as our repository has `hacktoberfest` in topics. `hacktoberfest accepted` is for PRs that might not get merged within this month's end but still would need to count towards their hacktoberfest contribution.

**Comment by pg99696** [comment]
> > @kbdharun, can you please add label 'hactoberfest-accepted' on PR?
> 
> This PR will automatically get reflected in your profile without the accepted label as our repository has `hacktoberfest` in topics. `hacktoberfest accepted` is for PRs that might not get merged within this month's end but still would need to count towards their hacktoberfest contribution.

Okay, got it. Thanks.

**Pull request merged**